spec/spec_helper.rb

```diff
@@ -49,6 +49,7 @@ def fixture_setup
   @druid = "druid:#{@obj}"
   @data = @fixtures.join("data", @obj)
   @derivatives = @fixtures.join('derivatives')
+  @derivatives2 = @fixtures.join('derivatives2')
```

Comment:
Please avoid setting instance vars like this, I know this is what has been done in the past, but the idiomatic way to do this in rspec is to use a `let` declaration just in the contexts where it's needed.